### DISCUSSION TITLE
Mais opções de escolhas

### DISCUSSION THREAD

**Discussion by sitariom** [YuriAlessandro/gerador-ficha-tormenta20#305] [Ideias]
Primeiramente, obrigado pelo ótimo trabalho. O gerador tem ajudado muito nas minhas sessões. Se me permite algumas sugestões. Seria interessante permitir que o usuário também escolha a devoção do personagem e a origem. Outra opção interessante seria permitir ou não poderes da tormenta e repetição de poderes. Muito obrigado e parabéns pelo trabalho

**Comment by YuriAlessandro** [comment]
As duas primeiras sugestões são bem legais, acho que dá pra gente encaixar (permitir que o usuário também escolha a devoção do personagem e a origem).

Quanto aos poderes, a gente ainda nem terminou tudo que gostaria de fazer com eles (aplicar os efeitos automaticamente, por exemplo), então acho que ficaria bem para depois. Mas vou deixar anotado aqui nas coisas a fazer.

**Comment by YuriAlessandro** [comment]
Isso foi adicionado na versão 2.2

**Comment by sitariom** [comment]
Ficou excelente!
Parabéns pelo trabalho. Já vou usar na minha mesa

Em qui, 9 de dez de 2021 20:41, Yuri Alessandro Martins <
***@***.***> escreveu:
